CFC_DataCollector/recommender/sample_pipeline.py

```diff
@@ -0,0 +1,373 @@
+from pymongo import MongoClient
+import logging
+from datetime import datetime
+
+import sys
+import os
+
+# On the server, we've installed miniconda for now, so we are just going to add
+# it to the python path
+sys.path.append("/home/ubuntu/miniconda/lib/python2.7/site-packages/")
```

Comment:
Isn't this just the same as the current mode inference pipeline? Is there a reason that this needs to be copied here?